Docs/Workflows.md

```diff
@@ -8,6 +8,7 @@ These workflows can be invoked from a step from any other repository's workflow.
 - Workflows with a `cancel-workflow-on-failure` parameter will by default cancel all jobs in the workflow run when the given reusable workflow fails (to save computing resources). You can disable this by setting the parameter to `"false"`.
 - To add the workflows to a project create a folder in the root of the repository that will call them, e.g. _.github/workflows/build-and-test.yml_ and/or _.github/workflows/publish-nuget.yml_. The examples below are for such files.
 - If you use these workflows with a self-hosted runner, then you'll need to fork this repository under the organization while keeping [these rules](https://docs.github.com/en/actions/using-workflows/reusing-workflows#access-to-reusable-workflows) in mind. Then, you need to enable workflows for the fork under its Actions tab (you'll see a big button for this). If you don't do the latter step, you'll get a "workflow was not found" error. Then you can also disable the `spelling-this-repo` and `validate-this-gha-refs` workflows, not to run them unnecessarily if you sync the fork with the original repo.<!--#spell-check-ignore-line-->
+- Some of the long-running workflows (for example solution builds and spell-checking) also support the `cancel-in-progress-for-this-pr` parameter. When it's set to `'true'`, will cancel the already running workflow run or job (e.g., if the job uses strategy matrix to run on different machine types). Default is `'false'`.
```

Comment:
When will the workflow be canceled and when just the job, exactly? E.g., what will happen with a workflow like OSOCE's, that calls the `build-and-test-orchard-core` and `spelling` reusable workflows, where the former uses a matrix? Should one only use these configs, or is there any scenario (when only calling such reusable workflows from your own workflow) when you'd put `cancel-in-progress: true` in your own workflow instead? Add this to the docs too.

Reply:
What I wrote was unclear, because cancellation is applied to the jobs only, so I simplified what's described here. There are multiple different cases and situations to consider, but I don't think this is the place to explain them all. However, the workflows that include this parameter supply their own explanation as in-line comments at the concurrency configuration (including how and why it's configured at the workflow or job level).